examples/getting-started-session-based/01-ETL-with-NVTabular.ipynb

```diff
@@ -104,7 +104,7 @@
    "metadata": {},
```

Comment:
<p>I think we should use Categorify with start_index ? In my understanding, we shift, because we do not have null values. But I think a lot of users will just copy the code and miss the information</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>